website/docs/r/schema_template_anp_epg_useg_attr.html.markdown

```diff
@@ -13,8 +13,9 @@ Resource for MSO Schema Template Application Network Profiles Endpoint Groups Us
 ## Example Usage ##
 
 ```hcl
+
 resource "mso_schema_template_anp_epg_useg_attr" "useg_attrs" {
-  schema_id     = "5eafca7d2c000052860a2902"
+  schema_id     = mso_schema.schema1.id
   anp_name      = "sanp1"
   epg_name      = "nkuseg"
```

Comment:
should refer to mso_schema_template_anp_epg

Reply:
ok